meinberlin/apps/plans/assets/FilterNav.jsx

```diff
@@ -56,6 +56,12 @@ class FilterNav extends React.Component {
     })
   }
 
+  closeSecondaryFilters () {
+    this.setState({
+      displaySecondaryFilters: !this.state.displaySecondaryFilters
```

Comment:
if it's just to close it, I would set it to false to make sure it's always doing that